uni/README.md

```diff
@@ -16,6 +16,15 @@ The token is read from the file assets/env/env.json, which you may need to creat
 }
 ```
 
+### Automated formatting
+
+In order to contribute, you must format you files using `dart format` manually or formatting on save using your IDE automatically, or you can install the git pre-commit hook doing the following command at the root of the directory that automatically formats staged files:
```

Comment:
Can you be more explicity where to run the command?
Many people open uni subfolder instead of repository folder and they may try to run on that folder

Reply:
We should also ponder whether it is more friendly to put the file at the root or outside the uni directory

Reply:
It should be in the directory root where .github lives

Reply:
It made more sense to add it here, since it is more technical in the uni subfolder

Reply:
But i can move it to the main readme if it's more friendly there

Reply:
I think it's ok as it is then

Reply:
@LuisDuarte1 The description have space to be more clear. Also you have a "you files" that should be "your files".
Sorry to be anoying here

Reply:
> @LuisDuarte1 The description have space to be more clear. Also you have a "you files" that should be "your files". Sorry to be anoying here

Done! tell me what you think now @thePeras

Reply:
Good! Do we also want to mention that in order to disable, we need to delete the hook?